support/COMMUNITY_CONTACTS.md

```diff
@@ -36,12 +36,12 @@ community contact's duty (subject to change) is as followed:
   mailing list.
 - Add your self as a member of Slack user group `@serving-help`
 - Check Slack channel
-  [#serving-questions](https://knative.slack.com/archives/C0186KU7STW) for
+  [#knative-serving](https://cloud-native.slack.com/archives/C04LMU0AX60) for
   unanswered questions. Any questions that relates to usability please instruct
   user to
   [open an usablity issue](https://github.com/knative/ux/issues/new?assignees=&labels=kind%2Ffriction-point&template=friction-point-template.md&title=)
   and to join the channel
-  [#user-experience](https://knative.slack.com/archives/C01JBD1LSF3) to capture
+  [#user-experience](https://slack.cncf.io/) to capture
```

Comment:
Is there going to be a user-experience channel? I dont see one right now.

Reply:
yes there is no user experience channel yet, should i remove this altogether?

Reply:
Docs and UX use the same channel, so you can point them to `#knative-documentation` / https://cloud-native.slack.com/archives/C04LY5G9ED7

Reply:
(the two working groups merged sometime last year)